src/Mvc/Mvc.ViewFeatures/src/Rendering/CheckBoxHiddenInputRenderMode.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Mvc.Rendering
```

Comment:
Suggest this should be in the same namespace as `HtmlHelperOptions`: `Microsoft.AspNetCore.Mvc.ViewFeatures` but defer to @pranavkm here

Reply:
Thanks for the quick response @dougbu. I initially put it in the `.Rendering` namespace to follow suit with `Html5DateRenderingMode` and `TagRenderMode`, but will also defer to @pranavkm.